tutorials/horizontally-scale-mysql-database-backend-with-google-cloud-sql-and-proxysql/index.md

```diff
@@ -0,0 +1,325 @@
+	```
+
+	Replace `[IP_ADDRESS]` with the IPv4 address of your Cloud SQL instance.
+
+3. Changes made via ProxySQL admin interface are stored in an in-memory database and do not take effect automatically. Run the commands below to bring the new changes into effect and save them to the disk:
+
+	```
+	LOAD MYSQL SERVERS TO RUNTIME;
+	SAVE MYSQL SERVERS TO DISK;
+	```
+
+4. Add the user credentials for connecting with this SQL query:
+
+	```
+	INSERT INTO mysql_users(username, password) VALUES (‘[USERNAME]’,’[PASSWORD]’);
+	```
+
+	Replace `[USERNAME]` and `[PASSWORD]` with the username and password of your Cloud SQL instance respectively. You may use the default user `root` and the root password you set when creating the Cloud SQL instance. For better security, create a new user with proper privileges and hostname restrictions. See [Creating and Managing MySQL Users](https://cloud.google.com/sql/docs/mysql/create-manage-users) and [MySQL Users](https://cloud.google.com/sql/docs/mysql/users) for more information.
+
+5. Similarly, load the new user into runtime and save it to disk with:
+
+	```
+	LOAD MYSQL USERS TO RUNTIME;
+	SAVE MYSQL USERS TO DISK;
+	```
+
+## Setting up Monitoring
+
+ProxySQL periodically check all the configured MySQL servers. To set up monitoring, [create a new user in your Cloud SQL instance](https://cloud.google.com/sql/docs/mysql/create-manage-users) and update the settings in ProxySQL admin interface:
+
+```
+SET mysql-monitor_username = '[USERNAME]';
+SET mysql-monitor_password = '[PASSWORD]';
+```
+
+Replace `[USERNAME]` and `[PASSWORD]` with the values of your own.
+
+If you would like to disable monitoring, run the following command in ProxySQL admin interface:
+
+```
+SET mysql-monitor_enabled = 'false';
+```
+
+After making changes, save the settings with:
+
+```
+LOAD MYSQL VARIABLES TO RUNTIME;
+SAVE MYSQL VARIABLES TO DISK;
+```
+
+See [ProxySQL documentation](https://github.com/sysown/proxysql/wiki/Monitor-Module) for more information on monitoring.
+
+## Configuring Your Application to Use ProxySQL
+
+In this tutorial, ProxySQL is installed in the same server as your application and accepts connection at `127.0.0.1`, port `6033`. Update the database configuration of your application accordingly to start using ProxySQL as a query router. Currently, since there is only one database server listed in ProxySQL, all the queries are routed there.
+
+## Creating Read Replicas
+
+Note: The following steps use [Cloud Console](https://console.cloud.google.com/sql) to create a Cloud SQL read replica. You can also create a read replica [via Cloud SDK](https://cloud.google.com/sdk/gcloud/reference/sql/instances/create) or [using the Cloud SQL API](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/). 
+
+1. Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances) in the Cloud Console. Click the instance you just created. Select the **Replicas** tab.
```

Comment:
You should specify that they click on the Instance ID to get to the Instance Details page where they can find the Replicas tab.